[0m20:46:31.866245 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b83edd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b8b36d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b8b3d10>]}


============================== 20:46:31.868760 | 8e6366f2-4363-404b-8ecd-d7cae6a8aeab ==============================
[0m20:46:31.868760 [info ] [MainThread]: Running with dbt=1.9.4
[0m20:46:31.869058 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/milosz.bertman/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt init', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:46:52.688138 [debug] [MainThread]: Starter project path: /Users/milosz.bertman/Projects/fannilla-data-dev/venv/lib/python3.11/site-packages/dbt/include/starter_project
[0m20:46:52.696690 [info ] [MainThread]: 
Your new dbt project "fannilla_data" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m20:46:52.697020 [info ] [MainThread]: Setting up your profile.
[0m20:48:42.965301 [info ] [MainThread]: Profile fannilla_data written to /Users/milosz.bertman/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m20:48:42.987440 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 131.13947, "process_in_blocks": "0", "process_kernel_time": 0.367369, "process_mem_max_rss": "188399616", "process_out_blocks": "0", "process_user_time": 1.154812}
[0m20:48:42.988888 [debug] [MainThread]: Command `dbt init` succeeded at 20:48:42.988712 after 131.14 seconds
[0m20:48:42.990056 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b83eb50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b8b3b90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b8928d0>]}
[0m20:48:42.990678 [debug] [MainThread]: Flushing usage events
[0m20:48:43.396886 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m20:48:52.104048 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b146850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b051810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b2b3d90>]}


============================== 20:48:52.106715 | 67c8f9d8-4376-4a42-a8d1-3f34b1ef0d61 ==============================
[0m20:48:52.106715 [info ] [MainThread]: Running with dbt=1.9.4
[0m20:48:52.107016 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/Users/milosz.bertman/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m20:48:52.117553 [info ] [MainThread]: dbt version: 1.9.4
[0m20:48:52.117824 [info ] [MainThread]: python version: 3.11.12
[0m20:48:52.118043 [info ] [MainThread]: python path: /Users/milosz.bertman/Projects/fannilla-data-dev/venv/bin/python3.11
[0m20:48:52.118214 [info ] [MainThread]: os info: macOS-15.4.1-arm64-arm-64bit
[0m20:48:52.643769 [info ] [MainThread]: Using profiles dir at /Users/milosz.bertman/.dbt
[0m20:48:52.644081 [info ] [MainThread]: Using profiles.yml file at /Users/milosz.bertman/.dbt/profiles.yml
[0m20:48:52.644259 [info ] [MainThread]: Using dbt_project.yml file at /Users/milosz.bertman/Projects/fannilla-data-dev/dbt_project.yml
[0m20:48:52.644391 [info ] [MainThread]: adapter type: bigquery
[0m20:48:52.644519 [info ] [MainThread]: adapter version: 1.9.1
[0m20:48:52.644649 [info ] [MainThread]: Configuration:
[0m20:48:52.644765 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m20:48:52.644878 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m20:48:52.644992 [info ] [MainThread]: Required dependencies:
[0m20:48:52.645141 [debug] [MainThread]: Executing "git --help"
[0m20:48:52.664792 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m20:48:52.665383 [debug] [MainThread]: STDERR: "b''"
[0m20:48:52.665563 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m20:48:52.665728 [info ] [MainThread]: Connection:
[0m20:48:52.665951 [info ] [MainThread]:   method: service-account
[0m20:48:52.666064 [info ] [MainThread]:   database: inance-capital
[0m20:48:52.666177 [info ] [MainThread]:   execution_project: inance-capital
[0m20:48:52.666294 [info ] [MainThread]:   schema: analytics
[0m20:48:52.666400 [info ] [MainThread]:   location: EU
[0m20:48:52.666502 [info ] [MainThread]:   priority: interactive
[0m20:48:52.666608 [info ] [MainThread]:   maximum_bytes_billed: None
[0m20:48:52.666722 [info ] [MainThread]:   impersonate_service_account: None
[0m20:48:52.666825 [info ] [MainThread]:   job_retry_deadline_seconds: None
[0m20:48:52.666930 [info ] [MainThread]:   job_retries: 1
[0m20:48:52.667039 [info ] [MainThread]:   job_creation_timeout_seconds: None
[0m20:48:52.667277 [info ] [MainThread]:   job_execution_timeout_seconds: 300
[0m20:48:52.667475 [info ] [MainThread]:   timeout_seconds: 300
[0m20:48:52.667598 [info ] [MainThread]:   client_id: None
[0m20:48:52.667716 [info ] [MainThread]:   token_uri: None
[0m20:48:52.667818 [info ] [MainThread]:   dataproc_region: None
[0m20:48:52.667923 [info ] [MainThread]:   dataproc_cluster_name: None
[0m20:48:52.668023 [info ] [MainThread]:   gcs_bucket: None
[0m20:48:52.668122 [info ] [MainThread]:   dataproc_batch: None
[0m20:48:52.668468 [info ] [MainThread]: Registered adapter: bigquery=1.9.1
[0m20:48:52.753415 [debug] [MainThread]: Acquiring new bigquery connection 'debug'
[0m20:48:52.753846 [debug] [MainThread]: On debug: select 1 as id
[0m20:48:52.754033 [debug] [MainThread]: Opening a new connection, currently in state init
[0m20:48:56.126240 [debug] [MainThread]: BigQuery adapter: https://console.cloud.google.com/bigquery?project=inance-capital&j=bq:EU:ce4a5c97-4210-4bf9-8848-11d3b244bb64&page=queryresults
[0m20:48:56.452043 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m20:48:56.452363 [info ] [MainThread]: [31m1 check failed:[0m
[0m20:48:56.452515 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/milosz.bertman/Projects/fannilla-data-dev/dbt_project.yml> not found

[0m20:48:56.455537 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 4.3888254, "process_in_blocks": "0", "process_kernel_time": 0.27405, "process_mem_max_rss": "195690496", "process_out_blocks": "0", "process_user_time": 1.120922}
[0m20:48:56.455855 [debug] [MainThread]: Command `dbt debug` failed at 20:48:56.455795 after 4.39 seconds
[0m20:48:56.456042 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m20:48:56.456222 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b242310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10311e190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x124eb8150>]}
[0m20:48:56.456432 [debug] [MainThread]: Flushing usage events
[0m20:48:56.833797 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m21:08:03.424783 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x115590a50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x115593090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1155b3ed0>]}


============================== 21:08:03.427586 | 1b5a6b38-9348-417f-bcd5-cb09c75a871f ==============================
[0m21:08:03.427586 [info ] [MainThread]: Running with dbt=1.9.4
[0m21:08:03.427890 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/milosz.bertman/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt debug --config-dir', 'send_anonymous_usage_stats': 'True'}
[0m21:08:03.438189 [info ] [MainThread]: To view your profiles.yml file, run:

open /Users/milosz.bertman/.dbt
[0m21:08:03.440161 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": true, "command_wall_clock_time": 0.053260583, "process_in_blocks": "0", "process_kernel_time": 0.143006, "process_mem_max_rss": "102285312", "process_out_blocks": "0", "process_user_time": 0.607347}
[0m21:08:03.440434 [debug] [MainThread]: Command `dbt debug` succeeded at 21:08:03.440374 after 0.05 seconds
[0m21:08:03.440653 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1155b3910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1155b2e10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1045920d0>]}
[0m21:08:03.440857 [debug] [MainThread]: Flushing usage events
[0m21:08:03.864184 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m21:22:54.121709 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107c8bf90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10796b850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107cb3f90>]}


============================== 21:22:54.130517 | 127b9e3a-0897-4afb-a4eb-f5417ccdf6ef ==============================
[0m21:22:54.130517 [info ] [MainThread]: Running with dbt=1.9.4
[0m21:22:54.130871 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/Users/milosz.bertman/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt deps', 'send_anonymous_usage_stats': 'True'}
[0m21:22:54.132326 [error] [MainThread]: Encountered an error:
Runtime Error
  dbt_project.yml does not parse to a dictionary
[0m21:22:54.134625 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_success": false, "command_wall_clock_time": 0.05811371, "process_in_blocks": "0", "process_kernel_time": 0.133922, "process_mem_max_rss": "102760448", "process_out_blocks": "0", "process_user_time": 0.620437}
[0m21:22:54.134967 [debug] [MainThread]: Command `dbt deps` failed at 21:22:54.134900 after 0.06 seconds
[0m21:22:54.135209 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x100b6e010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107c8dd50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107cb3110>]}
[0m21:22:54.135420 [debug] [MainThread]: Flushing usage events
[0m21:22:54.542772 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m21:23:12.265385 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110b429d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110bb39d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110b92cd0>]}


============================== 21:23:12.268404 | 3c6fb851-66a0-4566-83ea-10053987baeb ==============================
[0m21:23:12.268404 [info ] [MainThread]: Running with dbt=1.9.4
[0m21:23:12.268907 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/milosz.bertman/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt deps', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m21:23:12.269440 [error] [MainThread]: Encountered an error:
Runtime Error
  dbt_project.yml does not parse to a dictionary
[0m21:23:12.271189 [debug] [MainThread]: Resource report: {"command_name": "deps", "command_success": false, "command_wall_clock_time": 0.038908083, "process_in_blocks": "0", "process_kernel_time": 0.12319, "process_mem_max_rss": "103907328", "process_out_blocks": "0", "process_user_time": 0.609466}
[0m21:23:12.271425 [debug] [MainThread]: Command `dbt deps` failed at 21:23:12.271372 after 0.04 seconds
[0m21:23:12.271607 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1044e6450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110bb3e50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110bb3910>]}
[0m21:23:12.271790 [debug] [MainThread]: Flushing usage events
[0m21:23:12.749282 [debug] [MainThread]: An error was encountered while trying to flush usage events
